app/core/utils.js

```diff
@@ -1558,7 +1558,11 @@ const createLevelNumberMap = function (levels) {
     } else {
       practiceLevelCurrentCount = 0
     }
-    levelNumberMap[level.key] = levelNumber
+    if (level.key) {
```

Comment:
when will this be null/undefined?

Reply:
i don't know but ozaria has some levels has no original . maybe those `Intro`